docs/contributors/proposals.md

```diff
@@ -0,0 +1,27 @@
+# Minikube Proposals
+This document contains proposed functionality and rough design docs for said
+functionality.
+
+## OS X Menubar App 
+The OS X Menubar App (proposed here: https://github.com/kubernetes/minikube/issues/1866) 
+is designed to simplify basic minikube and k8s tasks for end users of minikube 
+installations who are not comfortable with complex command line options. The 
+app will sit in the menubar and allow a user easily accomplish the following 
+tasks: 
+
+* Start and Stop minikube (and show status of minikube/localkube)
+* Launch the k8s dashboard
+* See a list of pods and their status (red or green)
+* SSH into any pod
```

Comment:
Is this going to be exposed from the menu bar?  At least for the first pass, I wouldn't include anything outside the scope of the minikube or anything that overlaps too much with kubectl functionality.